website/meta/universe.json

```diff
@@ -1,5 +1,72 @@
 {
     "resources": [
+        {
+            "id": "scrubadub_spacy",
+            "title": "scrubadub_spacy",
+            "category": ["pipeline"],
+            "tags": [""],
```

Comment:
```suggestion
```

If there are no tags, we can just leave them out. The tags are not currently used in the front-end, it was just an initial idea for better clustering later on.

Reply:
Fix: Removed the tag line